cmd/kube-spawn/create.go

```diff
@@ -51,6 +51,9 @@ func init() {
 	createCmd.Flags().String("kubernetes-source-dir", "", "Path to directory with Kubernetes sources")
 	createCmd.Flags().String("hyperkube-image", "", "Kubernetes hyperkube image to use (if unset, upstream k8s is installed)")
 	createCmd.Flags().String("cni-plugin-dir", "/opt/cni/bin", "Path to directory with CNI plugins")
+	createCmd.Flags().String("cni-model", "weave", "CNI model to use (weave, flannel)")
```

Comment:
Usually they are called CNI plugins, not models. So I think `cni-plugin` would be better fitting option name and plugin should be used throughout, to stick to Kubernetes nomenclature

`calico` is missing from the list of possible values in the description.

Same for the option in the start subcommand.